boilerplates/react-union-boilerplate-liferay-basic/liferay/content-tag-portlet/src/main/java/eu/reactunion/boilerplate/portlet/ContentReactWidgetSupport.java

```diff
@@ -0,0 +1,70 @@
+package eu.reactunion.boilerplate.portlet;
+
+import javax.portlet.RenderRequest;
+
+import com.liferay.portal.kernel.module.configuration.ConfigurationException;
+import com.liferay.portal.kernel.theme.PortletDisplay;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.WebKeys;
+import eu.reactunion.boilerplate.configuration.ContentConfiguration;
+import eu.reactunion.boilerplate.portlet.data.InitData;
+import eu.reactunion.boilerplate.portlet.data.Messages;
+import org.osgi.service.component.annotations.Component;
+
+import eu.lundegaard.reactunion.support.ReactWidgetSupport;
+
+/**
+ * Implementation of {@link ReactWidgetSupport} which returns Init data from portlet instance configuration.
+ *
+ * @author Roman Srom (roman.srom@lundegaard.eu)
+ */
+@Component(
+        immediate = true,
+        service = ContentReactWidgetSupport.class
+)
+public class ContentReactWidgetSupport extends ReactWidgetSupport {
+
+//    @Override
```

Comment:
As mentioned above, no commented blocks of code, please :)

Reply:
It shows another usage of the react-union-support library, in my opinion it should be there.

Reply:
Oh, alright! If that's the case, maybe a comment explaining when to use this method instead of the default implementation would be helpful 🙂

Reply:
Documentation added.